app/src/main/java/luci/sixsixsix/powerampache2/presentation/main/MainViewModel.kt

```diff
@@ -228,22 +312,23 @@ class MainViewModel @Inject constructor(
         }
     }
 
-    // TODO USE worker
     private fun downloadSong(song: Song) = viewModelScope.launch {
         songsRepository.downloadSong(song).collect { result ->
```

Comment:
fix downloadSong, maybe should not be a flow since it only launches the worker, I don't need any result from this function